packages/blue-sdk/src/addresses.ts

```diff
@@ -544,6 +544,28 @@ const _addressesRegistry = {
     wNative: "0x3bd359C1119dA7Da1D913D1C4D2B7c461115433A",
     usdc: "0x754704Bc059F8C67012fEd69BC8A327a5aafb603",
   },
+  [ChainId.StableMainnet]: {
+    morpho: "0xa40103088A899514E3fe474cD3cc5bf811b1102e",
+    bundler3: {
+      bundler3: "0xA0bb114F927dF03d9a1a639b9c71F71B0FaFDf1B",
+      generalAdapter1: "0x59b1F4376a81e39c466A0A218447E4D36f39A96b",
+    },
+    adaptiveCurveIrm: "0x41e846FC8108b8527C1D4EDB4c9564E56442940f",
+    publicAllocator: "0xbCB063D4B6D479b209C186e462828CBACaC82DbE",
+    metaMorphoFactory: "0xb4ae5673c48621189E2bEfBA96F31912032DD1AE",
+    vaultV2Factory: "0x7fc35488803D49D00a94b206A223f7661898BE3a",
+    morphoMarketV1AdapterFactory: "0x2A5F218FE4Dac3b1f4E096e8ae83074bB1713833",
+    morphoVaultV1AdapterFactory: "0x4EF83ACD552598a1196c1aBDD0bA2EdE6f2237B4",
+    registryList: "0xCe93fcB2849EB886F1e81d45D2747dF803f843C3",
+    chainlinkOracleFactory: "0xF24C6eAB91e43EacE18a4e893a48565C09132505",
+    preLiquidationFactory: "0x741A6604e974FeAF35a5FBb1416B3e01c33e5C0e",
+
+    /**
+     * @deprecated This address is not the wrapped native address.
+     * It is a placeholder since native token wrapping is not supported by bundler3 on this chain
+     */
+    wNative: "0x0000000000000000000000000000000000000001",
```

Comment:
```suggestion
    preLiquidationFactory: "0x741A6604e974FeAF35a5FBb1416B3e01c33e5C0e",
```

Reply:
Exploring the commit history, I see you did remove it in https://github.com/morpho-org/sdks/pull/463/commits/35437bd6d7b8078c528a1af6ddcbef48f34a39b8 but for some reason you brought it back 2 commits later and i dont see any comment from Oumar or I asking for it to be brought back

We just chatted and we didnt know we could get rid of it. We only saw that it was first defined 0x000...001 then the official wNative address and then back to 0x01